packages/ui/components/LocalizedField/LocalizedField.tsx

```diff
@@ -96,7 +96,7 @@ export const LocalizedField = <T extends StrapiModel>({
           const langField = {
             ...field,
             name: `${field.name as string}_${lang}` as keyof T,
-            type: (field as FormLocalizedFields).subtype,
+            type: (field as FormLocalizedFields).type,
```

Comment:
there is an issue here. field allready have type. no need to set here. i did that to prevent dynamicfield call localizedfield.  instead of 'type' we must set localized: false.
```suggestion
            localized: false
```